content/en/docs/languages/java/_index.md

```diff
@@ -6,7 +6,7 @@ description: >-
 aliases: [/java, /java/metrics, /java/tracing]
 cascade:
   vers:
-    instrumentation: 2.0.0
+    instrumentation: 1.32.1
```

Comment:
Hm... seems like we need to fix the script to only update the 2.x version?

Reply:
hopefully we won't be making too many 1.x patches 😅

Reply:
So no need to fix that script, just close the PRs if they come up?

Reply:
thx